src/cli-constant.js

```diff
@@ -219,6 +219,13 @@ const detailedOptions = normalizeDetailedOptions({
     forwardToApi: true,
     description: "The line length where Prettier will try wrap."
   },
+  "prose-wrap": {
+    type: "boolean",
+    category: CATEGORY_FORMAT,
+    forwardToApi: true,
+    description: "Wrap the prose if it exceeds the print width.",
```

Comment:
I think we can remove the "the" here- "Wrap prose if it exceeds the print width".